<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersListDialog</class>
 <widget class="QDialog" name="UsersListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="i_userL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Пользователь:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="usersCB"/>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="closePB">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="delAllPB">
     <property name="text">
      <string>Сбросить всех пользователей</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="delUserPB">
     <property name="text">
      <string>Удалить пользователя</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
